apps/sr-frontend/src/app/App.tsx

```diff
@@ -29,7 +29,7 @@ export default function App() {
               <Stack gap={1} pt={1}>
                 <CharacterSelector />
                 <Character />
-                <LightConeEditor />
+                <LightConeInventory />
```

Comment:
Add instead of replace
```suggestion
                <LightConeEditor />
                <LightConeInventory />
```

Reply:
Done! Thanks

Reply:
Still not seeing LightConeEditor in app.tsx